contribs/gnodev/pkg/dev/node_state_test.go

```diff
@@ -0,0 +1,189 @@
+package dev
+
+import (
+	"context"
+	"strconv"
+	"testing"
+
+	emitter "github.com/gnolang/gno/contribs/gnodev/internal/mock"
+	"github.com/gnolang/gno/contribs/gnodev/pkg/events"
+	"github.com/gnolang/gno/gno.land/pkg/gnoclient"
+	"github.com/gnolang/gno/gno.land/pkg/gnoland"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNodeMovePreviousTX(t *testing.T) {
```

Comment:
Tests are more fun in parallel

Reply:
Ahaha, I can't, not the one that implies running a full node.